nemo/collections/asr/data/audio_to_diar_label.py

```diff
@@ -139,6 +145,129 @@ def assign_frame_level_spk_vector(rttm_timestamps, round_digits, frame_per_sec,
         return fr_level_target
 
 
+def get_subsegments_to_timestamps(
+    subsegments: List[Tuple[float, float]], feat_per_sec: int = 100, max_end_ts: float = None, decimals=2
+):
+    """
+    Convert subsegment timestamps to scale timestamps by multiplying with the feature rate and rounding.
+    All `ts` related tensors are dimensioned as (N, 2), where N is the number of subsegments.
```

Comment:
Maybe add a simple example to showcase what this function does? I'm a bit confused about the relation between segment and subsegment

Reply:
Yes this could be very confusing, because for end-to-end models, subsegments are equivalent to frames.

Modular models (vad + clustering, etc) : subsegments (usually 0.5~1.0s) in speech segments (usually 2 s ~ 15 s)
End-to-end models: frames in a session audio (the whole session length is a segment)

Added the definitions, and examples to clarify this.